logic/metrics/metrics.go

```diff
@@ -20,6 +22,14 @@ func Collect(iface string, peerMap models.PeerMap) (*models.Metrics, error) {
 		return &metrics, err
 	}
 	defer wgclient.Close()
+
+	if runtime.GOOS == "darwin" {
```

Comment:
why is this in the MQ PR?

Reply:
found this issue while testing so added a fix

Reply:
it's fine then